diff --git a/BUILD b/BUILD
new file mode 100644
index 0000000..2afebb4
--- /dev/null
+++ b/BUILD
@@ -0,0 +1,29 @@
+Software required to build:
+
+	Borland Pascal 7 is requird to build the game and some tools used to create data files.
+
+	The Digital Mars D compiler is required to the conversation and log conversion tools.
+		http://digitalmars.com/d/1.0/index.html
+
+	Borland's BGI Toolkit is required for VGA256.BGI
+
+	Python is required for the fixnames.py script. This converts all files in a directory in lowercase.
+
+Wine/DOSBox setup:
+	The Makefile assumes that Borland Pascal has been installed to F:\BP and Ironseed is placed in S:\IRONSEED
+
+	The command "sudo echo 0 > /proc/sys/vm/mmap_min_addr" may be required to run DOS executables under Wine. Newer versions of Wine may invoke DOSBox execute the command.
+
+
+Borland Pascal 7 setup:
+	The directories use in Borland Pascal need to be configured.
+	Run BP and in the menu "Options -> Directories" configure the directories like below.
+		EXE & TPU directory:  s:\ironseed\code
+		Include directories:  s:\ironseed\code;s:\ironseed\dmp
+		Unit directories:     s:\ironseed\code;s:\ironseed\dmp;f:\bp\units;f:\bp\winunits
+		Object directories:   s:\ironseed\code;s:\ironseed\dmp
+		Resource directories: f:\bp\units;
+
+BGI Toolkit:
+	VGA256.BGI should be extracted from the toolkit. It should be then converted to a .OBJ file with BINOBJ.EXE included with Borland Pascal.
+	VGA256.OBJ should be place in ironseed/code
\ No newline at end of file
diff --git a/code/cargtool.pas b/code/cargtool.pas
index 2350b3c..62f3fdf 100644
--- a/code/cargtool.pas
+++ b/code/cargtool.pas
@@ -1,4 +1,20 @@
 unit cargtool;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Cargo/Creation unit for IronSeed
diff --git a/code/combat.pas b/code/combat.pas
index 9063e5a..c724a23 100644
--- a/code/combat.pas
+++ b/code/combat.pas
@@ -1,4 +1,20 @@
 unit combat;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Battle/Combat unit for IronSeed
diff --git a/code/comm.pas b/code/comm.pas
index bbbf125..b32e6a9 100644
--- a/code/comm.pas
+++ b/code/comm.pas
@@ -1,4 +1,20 @@
 unit comm;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Communication unit for IronSeed
@@ -14,7 +30,7 @@ unit comm;
 {$O+}
 interface
 
-uses data,amp;
+uses data;
 
 procedure conversewithcrew;
 procedure continuecontact(hail: boolean);
diff --git a/code/comm2.pas b/code/comm2.pas
index fc43695..dd093cb 100644
--- a/code/comm2.pas
+++ b/code/comm2.pas
@@ -1,4 +1,20 @@
 unit comm2;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Communication unit #2 for IronSeed
diff --git a/code/crew2.pas b/code/crew2.pas
index f4af393..e2ff7c3 100644
--- a/code/crew2.pas
+++ b/code/crew2.pas
@@ -1,4 +1,20 @@
 unit crew2;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Crew Manipulation unit 2 for IronSeed
diff --git a/code/crewgen.pas b/code/crewgen.pas
index 92b3c34..25ca02a 100644
--- a/code/crewgen.pas
+++ b/code/crewgen.pas
@@ -1,4 +1,21 @@
 program crewgen;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 {$M 4500,300000,300000}
 {$S-,L-,D-}
 
diff --git a/code/crewinfo.pas b/code/crewinfo.pas
index 1daf332..e7f32fa 100644
--- a/code/crewinfo.pas
+++ b/code/crewinfo.pas
@@ -1,4 +1,20 @@
 unit crewinfo;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Crew Manipulation unit for IronSeed
diff --git a/code/crewtick.pas b/code/crewtick.pas
index c4e67df..84cff22 100644
--- a/code/crewtick.pas
+++ b/code/crewtick.pas
@@ -1,4 +1,20 @@
 unit crewtick;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {$O+}
 
diff --git a/code/data.pas b/code/data.pas
index 290233f..860835b 100644
--- a/code/data.pas
+++ b/code/data.pas
@@ -1,4 +1,20 @@
 unit data;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Data unit for IronSeed
diff --git a/code/detsound.pas b/code/detsound.pas
index 991b970..5f8b75a 100644
--- a/code/detsound.pas
+++ b/code/detsound.pas
@@ -1,6 +1,22 @@
 program detectsoundinformation;
+(********************************************************************
+    This file is part of Ironseed.
 
-uses strings,crt,MCP,DET_SB,DET_PAS,DET_ARIA,DETGUS;
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
+{$I DSMI.INC}, strings,crt,MCP,DET_SB,DET_PAS,DET_ARIA,DETGUS;
 
 var
  scard: TSoundCard;
diff --git a/code/display.pas b/code/display.pas
index cdaa15f..a593cf6 100644
--- a/code/display.pas
+++ b/code/display.pas
@@ -1,4 +1,20 @@
 unit display;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Ship Display Control unit for IronSeed
diff --git a/code/emstool.pas b/code/emstool.pas
index 2612fb8..103bc73 100644
--- a/code/emstool.pas
+++ b/code/emstool.pas
@@ -1,4 +1,20 @@
 unit emstool;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 interface
 
@@ -154,4 +170,4 @@ end;
 
 begin
  ems.initialize;
-end.
\ No newline at end of file
+end.
diff --git a/code/ending.pas b/code/ending.pas
index 0f60004..e12599b 100644
--- a/code/ending.pas
+++ b/code/ending.pas
@@ -1,4 +1,21 @@
 unit ending;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 {***************************
 
   Endgame Sequence for IronSeed
diff --git a/code/explore.pas b/code/explore.pas
index 8a6db39..a6bd9ab 100644
--- a/code/explore.pas
+++ b/code/explore.pas
@@ -1,4 +1,20 @@
 unit explore;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Planet Exploration unit for IronSeed
diff --git a/code/gmouse.pas b/code/gmouse.pas
index cba494b..da4854e 100644
--- a/code/gmouse.pas
+++ b/code/gmouse.pas
@@ -1,4 +1,20 @@
 unit gmouse;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Mouse Utilities unit for IronSeed
diff --git a/code/heapchk.pas b/code/heapchk.pas
index fa8a21c..b88f426 100644
--- a/code/heapchk.pas
+++ b/code/heapchk.pas
@@ -1,4 +1,20 @@
 unit heapchk;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 interface
 procedure WriteHexWord(w: Word);
diff --git a/code/info.pas b/code/info.pas
index fec4879..f0c81f1 100644
--- a/code/info.pas
+++ b/code/info.pas
@@ -1,4 +1,20 @@
 unit info;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Information unit for IronSeed
diff --git a/code/init.pas b/code/init.pas
index babe559..95e635e 100644
--- a/code/init.pas
+++ b/code/init.pas
@@ -1,4 +1,20 @@
 unit init;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Initialization unit for IronSeed
diff --git a/code/install.pas b/code/install.pas
index 8d97f66..80eb6e7 100644
--- a/code/install.pas
+++ b/code/install.pas
@@ -1,4 +1,21 @@
 program installit;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 {$M 4000,0,0}
 {
   Installation Unit for IronSeed
diff --git a/code/intro.pas b/code/intro.pas
index 1f18067..4662c54 100644
--- a/code/intro.pas
+++ b/code/intro.pas
@@ -1,4 +1,21 @@
 program intro;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 {$M 5500,436000,436000}
 {$S-,D-,L-}
 
diff --git a/code/is.pas b/code/is.pas
index 622e55f..b835d51 100644
--- a/code/is.pas
+++ b/code/is.pas
@@ -1,4 +1,21 @@
 program ironseed;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 {$M 2600,0,0}
 {S-,D-,L-}
 
diff --git a/code/journey.pas b/code/journey.pas
index 40e7a1d..caa53e7 100644
--- a/code/journey.pas
+++ b/code/journey.pas
@@ -1,4 +1,20 @@
 unit journey;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Main Control unit for IronSeed
diff --git a/code/main.pas b/code/main.pas
index 43bc175..666c39c 100644
--- a/code/main.pas
+++ b/code/main.pas
@@ -1,4 +1,21 @@
 program main;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 {$M 6500,335000,655360} (*390000*)
 {$S-,L-,D-}
 
diff --git a/code/modplay.pas b/code/modplay.pas
index 638fa7b..3d71d54 100644
--- a/code/modplay.pas
+++ b/code/modplay.pas
@@ -1,4 +1,20 @@
 unit modplay;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {$O+}
 
diff --git a/code/saveload.pas b/code/saveload.pas
index 237e7c2..e545b07 100644
--- a/code/saveload.pas
+++ b/code/saveload.pas
@@ -1,4 +1,20 @@
 unit saveload;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Save/Load Game and Utility Unit for IronSeed
diff --git a/code/shipmake.pas b/code/shipmake.pas
index 84ceaea..bde1db8 100644
--- a/code/shipmake.pas
+++ b/code/shipmake.pas
@@ -1,4 +1,20 @@
 program makeship;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 type
  alienshiptype=
diff --git a/code/config.pas b/code/sndcfg.pas
similarity index 92%
rename from code/config.pas
rename to code/sndcfg.pas
index 31374a7..43b3f28 100644
--- a/code/config.pas
+++ b/code/sndcfg.pas
@@ -1,7 +1,24 @@
 program ConfigureFrinj;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 {$M 2000,0,0}
 
-uses crt, win, strings, dos;
+uses crt, strings, dos;
 
 type
    configfile = 
diff --git a/code/starter.pas b/code/starter.pas
index febae7f..cfcfc38 100644
--- a/code/starter.pas
+++ b/code/starter.pas
@@ -1,4 +1,20 @@
 unit starter;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Initialization for IronSeed
diff --git a/code/usecode.pas b/code/usecode.pas
index 5638079..9d41358 100644
--- a/code/usecode.pas
+++ b/code/usecode.pas
@@ -1,4 +1,20 @@
 unit usecode;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Ship Display Initialization unit for IronSeed
diff --git a/code/utils.pas b/code/utils.pas
index 930e8d0..42759d1 100644
--- a/code/utils.pas
+++ b/code/utils.pas
@@ -1,4 +1,20 @@
 unit utils;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    General Utilities for IronSeed
diff --git a/code/utils2.pas b/code/utils2.pas
index 2dfa859..05771f8 100644
--- a/code/utils2.pas
+++ b/code/utils2.pas
@@ -1,4 +1,20 @@
 unit utils2;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    Overlayable Utilites for IronSeed
diff --git a/code/version.pas b/code/version.pas
index 130bfd2..5aba570 100644
--- a/code/version.pas
+++ b/code/version.pas
@@ -1,4 +1,20 @@
 unit version;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 interface
 var
@@ -7,5 +23,5 @@ implementation
 begin
    versionstring :=
    {1234567890}
-   'v1.20.0016';
+   'v1.30.0001';
 end.
diff --git a/code/weird.pas b/code/weird.pas
index a1b7840..9d274fb 100644
--- a/code/weird.pas
+++ b/code/weird.pas
@@ -1,4 +1,20 @@
 unit weird;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 {***************************
    CatchAll unit for IronSeed
diff --git a/crealist.txt b/crealist.txt
deleted file mode 100644
index e69de29..0000000
diff --git a/makedata/aliemake.pas b/makedata/aliemake.pas
index 15b90ed..0047fa8 100644
--- a/makedata/aliemake.pas
+++ b/makedata/aliemake.pas
@@ -1,4 +1,20 @@
 program makealiencontacts;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 type
  alientype=
diff --git a/makedata/artimake.pas b/makedata/artimake.pas
index fa8b914..fb56452 100644
--- a/makedata/artimake.pas
+++ b/makedata/artimake.pas
@@ -1,4 +1,20 @@
 program makeartifact;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 type
  artifacttype=string[10];
diff --git a/makedata/cargmake.pas b/makedata/cargmake.pas
index 5cb33fe..ec6469b 100644
--- a/makedata/cargmake.pas
+++ b/makedata/cargmake.pas
@@ -1,4 +1,20 @@
 program generatecargodata;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 const
  maxcargo= 145;
@@ -16,9 +32,9 @@ var
  c: char;
 
 begin
- assign(f,'data\cargo.dta');
+ assign(f,'data/cargo.dta');
  rewrite(f);
- assign(ft,'makedata\cargo.txt');
+ assign(ft,'makedata/cargo.txt');
  reset(ft);
  readln(ft);
  for i:=1 to maxcargo do
diff --git a/makedata/creamake.pas b/makedata/creamake.pas
index ecca8b1..0b9ba50 100644
--- a/makedata/creamake.pas
+++ b/makedata/creamake.pas
@@ -1,4 +1,20 @@
 program generatecargodata;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 uses crt;
 
@@ -72,7 +88,7 @@ begin
  reset(f2);
  read(f2,cr^);
  close(f2);
- assign(ft,'itemdata.txt');
+ assign(ft,'other\itemdata.txt');
  rewrite(ft);
  writeln(ft,'    ITEM TO CREATE             PART #1             PART #2              PART#3 WORTH LEVELS            ');
  writeln(ft,'------------------ ------------------- ------------------- ------------------- ----- ------------------');
diff --git a/makedata/crewmake.pas b/makedata/crewmake.pas
index 690b5f6..0ca55b6 100644
--- a/makedata/crewmake.pas
+++ b/makedata/crewmake.pas
@@ -1,4 +1,21 @@
 program makecrew;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 uses crt;
 
 type
@@ -38,7 +55,7 @@ begin
  assign(fs,'makedata\crew.txt');
  reset(fs);
  assign(ft,'data\crew.dta');
- reset(ft);
+ rewrite(ft);
  clrscr;
  for a:=1 to 30 do
   begin
diff --git a/makedata/elemmake.pas b/makedata/elemmake.pas
index b23b7cb..7d96863 100644
--- a/makedata/elemmake.pas
+++ b/makedata/elemmake.pas
@@ -1,4 +1,20 @@
 program generatecargodata;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 type
  cargotype=
@@ -17,7 +33,7 @@ var
 
 begin
  assign(f,'\ironseed\data\elements.dta');
- reset(f);
+ rewrite(f);
  assign(ft,'\ironseed\makedata\element.txt');
  reset(ft);
  read(ft,index);
diff --git a/makedata/eventmak.pas b/makedata/eventmak.pas
index ce159d6..68b69ba 100644
--- a/makedata/eventmak.pas
+++ b/makedata/eventmak.pas
@@ -1,4 +1,20 @@
 program eventmake;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 uses crt;
 
diff --git a/makedata/getfont.pas b/makedata/getfont.pas
index 113b611..bc645c2 100644
--- a/makedata/getfont.pas
+++ b/makedata/getfont.pas
@@ -1,4 +1,20 @@
 program getfontfromfile;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 uses crt,graftool;
 
diff --git a/makedata/iconmake.pas b/makedata/iconmake.pas
index 8dec2c9..92cf1d1 100644
--- a/makedata/iconmake.pas
+++ b/makedata/iconmake.pas
@@ -1,4 +1,21 @@
 program convertplanicons;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
+
 uses crt, data;
 
 type
diff --git a/makedata/itemmake.pas b/makedata/itemmake.pas
index e1fff9e..16ad8e8 100644
--- a/makedata/itemmake.pas
+++ b/makedata/itemmake.pas
@@ -1,4 +1,20 @@
 program getiteminfostuff;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 type
  iteminfotype=
diff --git a/makedata/makename.pas b/makedata/makename.pas
index e26310d..63cb4fd 100644
--- a/makedata/makename.pas
+++ b/makedata/makename.pas
@@ -1,4 +1,20 @@
 program makenames;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 var
  i,j,a,b: integer;
diff --git a/makedata/scanmake.pas b/makedata/scanmake.pas
index 4cd82d9..1b4eea7 100644
--- a/makedata/scanmake.pas
+++ b/makedata/scanmake.pas
@@ -1,4 +1,20 @@
 program generatecargodata;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 type
  cargotype=
@@ -21,7 +37,7 @@ begin
  assign(ft,'\ironseed\makedata\scandata.txt');
  reset(ft);}
  assign(f,'data/scan.dta');
- reset(f);
+ rewrite(f);
  assign(ft,'makedata/scandata.txt');
  reset(ft);
  for i:=1 to 17 do
diff --git a/makedata/shipmake.pas b/makedata/shipmake.pas
index 13d10ef..b3b48a0 100644
--- a/makedata/shipmake.pas
+++ b/makedata/shipmake.pas
@@ -1,4 +1,20 @@
 program getshipdata;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 type
  alienshiptype=
diff --git a/makedata/sysmake.pas b/makedata/sysmake.pas
index 9cde194..ff7e057 100644
--- a/makedata/sysmake.pas
+++ b/makedata/sysmake.pas
@@ -1,4 +1,20 @@
 program displaysystems;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 
 uses crt;
@@ -21,11 +37,11 @@ var
 
 procedure display;
 begin
- assign(f,'data\sysname.dta');
+ assign(f,'data/sysname.dta');
  reset(f);
- assign(f2,'data\sysset.dta');
+ assign(f2,'data/sysset.dta');
  reset(f2);
- assign(ft,'sysdata.txt');
+ assign(ft,'other/sysdata.txt');
  rewrite(ft);
  for a:=1 to 250 do
   begin
@@ -58,11 +74,9 @@ end;
 
 
 begin
- display;
- exit;
- assign(ft,'makedata\names.txt');
+ assign(ft,'makedata/names.txt');
  reset(ft);
- assign(f,'data\sysname.dta');
+ assign(f,'data/sysname.dta');
  rewrite(f);
  textmode(co80);
  for a:=1 to 250 do
@@ -76,4 +90,7 @@ begin
   end;
  close(ft);
  close(f);
+
+ display;
+ exit;
 end.
\ No newline at end of file
diff --git a/makedata/weapmake.pas b/makedata/weapmake.pas
index 9efe8be..d163372 100644
--- a/makedata/weapmake.pas
+++ b/makedata/weapmake.pas
@@ -1,4 +1,20 @@
 program generateweapondata;
+(********************************************************************
+    This file is part of Ironseed.
+
+    Ironseed is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Ironseed is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Ironseed.  If not, see <http://www.gnu.org/licenses/>.
+********************************************************************)
 
 uses crt;
 
diff --git a/version.txt b/version.txt
index b111351..d76d053 100644
--- a/version.txt
+++ b/version.txt
@@ -173,3 +173,6 @@ v1.20.0016:
 
 
 	
+v1.30.0001:
+	Source, graphical assets and sound effects have been released under GPL.
+	The DMP library has been replaced with stubs.
